chromium/manifest.json

```diff
@@ -1,43 +1,43 @@
 {
     "author": {
         "email": "eff.software.projects@gmail.com"
-    }, 
+    },
     "background": {
         "scripts": [
-            "rules.js", 
-            "storage.js", 
-            "util.js", 
-            "background.js", 
+            "rules.js",
+            "storage.js",
+            "util.js",
+            "background.js",
             "incognito-cache-clearing.js"
         ]
-    }, 
+    },
     "browser_action": {
         "default_icon": {
             "38": "icons/icon-active-38.png"
-        }, 
-        "default_popup": "popup.html", 
+        },
+        "default_popup": "popup.html",
         "default_title": "__MSG_about_ext_name__"
-    }, 
-    "default_locale": "en", 
-    "description": "__MSG_about_ext_description__", 
-    "devtools_page": "devtools.html", 
-    "homepage_url": "https://www.eff.org/https-everywhere", 
+    },
+    "default_locale": "en",
+    "description": "__MSG_about_ext_description__",
+    "devtools_page": "devtools.html",
+    "homepage_url": "https://www.eff.org/https-everywhere",
     "icons": {
-        "128": "icons/icon-active-128.png", 
+        "128": "icons/icon-active-128.png",
         "48": "icons/icon-active-48.png"
-    }, 
-    "incognito": "spanning", 
-    "manifest_version": 2, 
-    "minimum_chrome_version": "51", 
-    "name": "__MSG_about_ext_name__", 
+    },
+    "incognito": "spanning",
+    "manifest_version": 2,
+    "minimum_chrome_version": "51",
+    "name": "__MSG_about_ext_name__",
     "permissions": [
-        "webNavigation", 
-        "webRequest", 
-        "webRequestBlocking", 
-        "tabs", 
-        "cookies", 
-        "storage", 
-        "<all_urls>"
-    ], 
+        "webNavigation",
+        "webRequest",
+        "webRequestBlocking",
+        "tabs",
+        "cookies",
+        "storage",
+        "*://*/*"
```

Comment:
A rewrite of the tools will require a lot of my attention to review, which I can't provide until after `webextensions` is deployed next month.